site/src/common/blocks/HeadingBlock.tsx

```diff
@@ -76,10 +76,21 @@ export const HeadingBlock = withPreview(
 
         return (
             <Root $textAlign={textAlignmentMap[textAlignment]}>
-                <Typography variant={"h400"} component={"h5"} bottomSpacing>
-                    <RichTextBlock data={eyebrow} renderers={eyebrowRenderers} />
-                </Typography>
-                <RichTextBlock data={headline} renderers={getHeadlineRenderers(headlineTag)} />
+                {hasRichTextBlockContent(eyebrow) && (
+                    <Typography variant={"h400"} component={"h5"} bottomSpacing>
+                        <RichTextBlock data={eyebrow} renderers={eyebrowRenderers} />
+                    </Typography>
+                )}
+                <PreviewSkeleton
+                    hasContent={hasRichTextBlockContent(headline)}
+                    title={
+                        <HeadlineSkeleton variant={"h550"} component={"span"}>
+                            Headline lorem Ipsum
```

Comment:
But I wouldn't add lorem ipsum here

```suggestion
                            Headline
```